_utilities/layout-grid.md

```diff
@@ -30,6 +28,63 @@ subnav:
   href: '#sass-mixins'
 - text: Latest updates
   href: '#changelog'
+spacing_settings:
+  - name: Small grid gap
+    subsection: grid-gap token
+    section: true
+    description: Sets `"grid-gap-sm"` token.
+    var: $theme-column-gap-sm
+    default: 2px
+    type: units
+  - name: Medium grid gap
+    subsection: grid-gap token
+    description: Sets `"grid-gap-md"` token.
+    var: $theme-column-gap-md
+    default: 2
+    type: units
+  - name: Large grid gap
+    subsection: grid-gap token
+    description: Sets `"grid-gap-lg"` token.
+    var: $theme-column-gap-lg
+    default: 3
+    type: units
+  - name: Mobile grid gap
+    subsection: grid-gap token
+    description: Sets value of `"grid-gap"` at `mobile` width.
+    var: $theme-column-gap-mobile
+    default: 2
+    type: units
+  - name: Desktop grid gap
+    subsection: grid-gap token
+    description: Sets value of `"grid-gap"` at `desktop` width.
```

Comment:
```suggestion
    description: Sets the value of the `.grid-gap` utility class at `desktop` window widths.
```

Reply:
Removed in [5b6e4da](https://github.com/uswds/uswds-site/pull/2276/commits/5b6e4da238ec15b95c66f14f23af8b3dab049315)